frontend/src/modules/Shared/index.js

```diff
@@ -0,0 +1,9 @@
+export * from './FeedComments';
+export * from './FeedCommentAdd';
+export * from './KeyValueTagList';
+export * from './KeyValueTagUpdateForm';
+export * from './ShareInboxList';
```

Comment:
`ShareInboxList` and `ShareInboxListItem` are pure dataset functionality. If datasets are off `listDatasetShareObjects` and `getShareRequestsToMe` will fail

Reply:
It works for me. Let's leave it there for a while. But we should not forget to return back to it when we modularize dataset shares.